### DISCUSSION TITLE
Remove address block from checkout block -- (store selling only digital products)

### DISCUSSION THREAD

**Discussion by evamusby** [woocommerce/woocommerce#44950] [WooCommerce, Blocks, and Site Editing]
Is there a way of removing from the checkout page the section requiring the customer's address?
If not, please make that a feature.
I only sell digital products and I'm sure this lengthy filling in of an address is a real turnoff for customers
Thank you

**Comment by nielslange** [comment]
Hello @evamusby 👋

Thanks for reaching out. I understand that you want to remove the address fields from the checkout when someone is purchasing a digital product. Currently, when buying a digital/virtual product, the shipping address gets hidden, but not the billing address. To clarify your request, you expected the billing address also to be hidden, leading to this result:

<img width="1348" alt="Screenshot 2023-10-06 at 17 40 13" src="https://github.com/woocommerce/woocommerce-blocks/assets/3323310/d9a49788-65de-423f-9e64-f7ce840c5eeb">

Can you confirm that this is what you are aiming for?

**Reply by evamusby** [reply]
Yes, you understand my question perfectly, thank you. Can it be done?

**Comment by passatgt** [comment]
This is a use-case in Hungary too. If you sell digital products, you can generate e-receipts instead of invoices and in this case you don't need to collect any billing details, just an e-mail address is enough. However, you need to provide an option to the customer if it wants an invoice instead of a receipt, in this case you need to collect the billing details. Currently i have a custom plugin that uses woocommerce_checkout_fields to show and hide the billing fields based on a checkbox(if checked, user needs an invoice, so the fields are shown again).

**Reply by evamusby** [reply]
Thank you. Is that your Woo Billingo Plus plugin? And that definitely works with Woocommerce blocks? 
It seems to do an lot of things I don't need (e.g. my customers don't need an option) and I wonder if there's some simple code I can put into functions.php instead?

**Reply by passatgt** [reply]
Yes, thats the plugin but it is not compatible with blocks yet, only regular checkout.

**Reply by evamusby** [reply]
Ah OK. thanks anyway. I had no problem disabling those address fields with the regular checkout system. It's the blocks system that seems difficult (or impossible?)
